assets/package.json

```diff
@@ -10,7 +10,8 @@
   "license": "ISC",
   "dependencies": {
     "@monaco-editor/react": "^4.4.5",
-    "@openfn/describe-package": "^0.0.8",
+    "@openfn/adaptor-docs": "file:../../kit/dist/openfn-adaptor-docs-0.0.1.tgz",
```

Comment:
Obviously these dependencies are wrong but I can't update them properly until the actual packages are released. For now, use the export utility in `kit` to test locally.